src/resources/startup.py

```diff
@@ -53,29 +53,34 @@
 
 def serialize_byte_tensor(input_tensor):
     """
-        Serializes a bytes tensor into a flat numpy array of length prepend bytes.
-        Can pass bytes tensor as numpy array of bytes with dtype of np.bytes_,
-        numpy strings with dtype of np.str_ or python strings with dtype of np.object.
-        Parameters
-        ----------
-        input_tensor : np.array
-            The bytes tensor to serialize.
-        Returns
-        -------
-        serialized_bytes_tensor : np.array
-            The 1-D numpy array of type uint8 containing the serialized bytes in 'C' order.
-        Raises
-        ------
-        InferenceServerException
-            If unable to serialize the given tensor.
-        """
+    Serializes a bytes tensor into a flat numpy array of length prepended bytes.
+    Can pass bytes tensor as numpy array of bytes with dtype of np.bytes_,
```

Comment:
Shouldn't we delete the "Can pass bytes ... of np.bytes_" since that will strip the trailing zeros. Instead:
"The numpy array should use dtype of np.object_. For np.str_ and np.bytes_, numpy will remove ... "

Reply:
You don't need to rerun CI